### PR TITLE
Fix/encrypt json rpc

### PR DESCRIPTION
**Pull request by maycon-mello**
There is a PR in the sdk as well.
https://github.com/docknetwork/react-native-sdk/pull/6/files

### CODE REVIEW

File: src/polkadot-test.js [hunk 1]
```diff
@@ -1,97 +1,130 @@
...
-  dock.setAccount(testAccount);
+function main() {
+    // const data = encrypt('test');
+    // const decrypted = decrypt(data);
+}
 
-  // const data = dock.api.tx.balances.transfer('37GfhtNUJk1aJhXuGxNJsAGenteDBX3DTVAvuBZm49Kqc9wA', '10000');
-  // const fee = await data.paymentInfo(testAccount);
+main();
+// import Keyring from '@polkadot/keyring';
```
Comment (mike-parkhill) [inline src/polkadot-test.js:31]: is there a plan to uncomment this code?

File: src/core/crypto.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
+import crypto from 'crypto';
+
+const algorithm = 'aes-256-cbc';
+
+export const DEFAULT_KEY = '776fe87eec8c9ba8417beda00b23cf22f5e134d9644d0a195cd9e0b7373760c1';
```
Comment (mike-parkhill) [inline src/core/crypto.js:6]: is there a plan to allow this key to be injected rather than hard-coding it?

Comment (maycon-mello) [reply src/core/crypto.js:6]: I can  setup some env variables on the sdk, let me check

File: src/polkadot-test.js (hunk 1 above)
Comment (maycon-mello) [reply src/polkadot-test.js:31]: I think we can remove this file, I've been using for some testing.

File: src/core/crypto.js [hunk 3]
```diff
@@ -3,9 +3,11 @@ import crypto from 'crypto';
 
 const algorithm = 'aes-256-cbc';
 
-export const DEFAULT_KEY = '776fe87eec8c9ba8417beda00b23cf22f5e134d9644d0a195cd9e0b7373760c1';
+export const DEFAULT_KEY = process.env.ENCRYPTION_KEY || process.env.REACT_APP_ENCRYPTION_KEY;
 export const SECURE_JSON_RPC = true;
 
+console.log(`encryption key`, DEFAULT_KEY);
```
Comment (mike-parkhill) [inline src/core/crypto.js:9]: probably shouldn't be logging this

Comment (maycon-mello) [reply src/core/crypto.js]: thanks

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**